src/databox/azext_databox/_client_factory.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def cf_databox(cli_ctx, *_):
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    from azext_databox.vendored_sdks.databox import DataBoxManagementClient
+    return get_mgmt_service_client(cli_ctx, DataBoxManagementClient)
+
+
+def cf_jobs(cli_ctx, *_):
+    return cf_databox(cli_ctx).jobs
+
+
+def cf_service(cli_ctx, *_):
```

Comment:
do we need this client factory?

Reply:
will remove it.

---
In reply to: [388072166](https://github.com/Azure/azure-cli-extensions/pull/1344#discussion_r388072166) [](ancestors = 388072166)